lib/katello/tasks/upgrades/4.8/fix_incorrect_providers.rake

```diff
@@ -0,0 +1,32 @@
+namespace :katello do
```

Comment:
Gotta add this task to https://github.com/Katello/katello/blob/master/db/seeds.d/111-upgrade_tasks.rb in order to get picked up by the installer